### PR TITLE
Add the blog for first 4 InKind sponsors

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @anistark @raukadah @vipulgupta2048

### CODE REVIEW

**Comment by anistark** [comment]
> I don't know where the sponsor images are.

I've added in my PR. Please refer to them and update the links accordingly.

**Review by raukadah — COMMENTED** [review]
few comments, rest looks good.

File: content/blog/2021/announcing-inkind-sponsors-first.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Title: Announcing some of our InKind sponsrs for PyCon India 2021 - DeepSource, EPAM, FulFil.IO, and IBM Quantum
+Date: 2021-08-25 17:00
+Category: 2021
+Slug: announce-inkind-sponsors
+Summary: We are happy to announce some of our InKind Sponsors for PyCon India 2021!
+
+
+The PyCon India team previously decided to have no sponsors for this year's conference. You can read more about that in our [announcement blog](https://in.pycon.org/blog/2021/pycon-india-announcement.html). 
+
+But, due to popular demand, we have decided to have some InKind sponsors to improve our attendees' experience of the conference from home.
```
[the remark this answers is not in the document]
Comment (Sc0rpi0n101) [reply content/blog/2021/announcing-inkind-sponsors-first.md:10]:
```suggestion
Due to popular demand, we have decided to have some InKind sponsors to improve our attendee experience of the conference from home. 
```

**Comment by anistark** [comment]
Since 1993, EPAM Systems, Inc. (NYSE: EPAM) has leveraged its software engineering expertise to become a leading global product development, digital platform engineering, and top digital and product design agency. Through its ‘Engineering DNA’ and innovative strategy, consulting, and design capabilities, EPAM works in collaboration with its customers to deliver next-gen solutions that turn complex business challenges into real business outcomes. EPAM’s global teams serve customers in more than 35 countries across North America, Europe, Asia and Australia. As a recognized market leader in multiple categories among top global independent research agencies, EPAM was one of only four technology companies to appear on Forbes 25 Fastest Growing Public Tech Companies list every year of publication since 2013 and ranked as the top IT services company on Fortune’s 100 Fastest-Growing Companies list in 2019 and 2020. Learn more at [welcome.epam.in](welcome.epam.in) and follow us on [Twitter](https://twitter.com/EPAM_India) and [LinkedIn](https://www.linkedin.com/company/epam-systems/about/)

@Sc0rpi0n101 EPAM gave this ^. Please update accordingly.

**Review by chkumar246 — APPROVED** [review]

**Review by raukadah — APPROVED** [review]

File: content/blog/2021/announcing-inkind-sponsors-first.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+Title: Announcing some of our InKind sponsrs for PyCon India 2021 - DeepSource, EPAM, FulFil.IO, and IBM Quantum
```
Comment (vipulgupta2048) [inline content/blog/2021/announcing-inkind-sponsors-first.md:1]:
```suggestion
Title: Announcing DeepSource, EPAM, FulFil.IO, and IBM Quantum as our In-Kind sponsors for PyCon India 2021 (Part-1)```

**Review by vipulgupta2048 — COMMENTED** [review]
The spellings. Check the spelling atleast.

**Comment by raukadah** [comment]
@palnabarun We need to deploy it today i.e Sep 6th 2021

**Pull request merged**